rome-modules/src/main/java/com/rometools/modules/mediarss/io/MediaModuleParser.java

```diff
@@ -152,7 +165,12 @@ public Module parse(final Element mmRoot, final Locale locale) {
 
                     try {
                         if (content.getAttributeValue("fileSize") != null) {
-                            mc.setFileSize(Long.valueOf(content.getAttributeValue("fileSize")));
+                            String fileSizeAttr = content.getAttributeValue("fileSize").replaceAll("\\s", "");
+                            if (fileSizeAttr.matches("\\d+")) {
```

Comment:
Maybe handle this case inside `parseFileSize`? For example, return `number` if `matcher.group(2) == null`.